src/index.html

```diff
@@ -1,17 +1,164 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta 
+     name="viewport"
+     content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+     class="form" 
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+     method="post"
+    >
+      <fieldset class="form__fieldset">
+        <legend>Personal information</legend>
+        <label class="form__field">
+          Surname:
+          <input 
+           type="text" 
```

Comment:
```suggestion
          <input 
            type="text" 
```
not fixed, should be 2 spaces, check in other places